### DISCUSSION TITLE
taro微信小程序插件开发-为插件页面指定抽象组件报错

### DISCUSSION THREAD

**Discussion by touristzn** [NervJS/taro#12903] [Q & A]
插件页面中：
list.config.ts：
"componentGenerics": {
    "mp-view": true,
  }
list.tsx: <mp-view></mp-view>

小程序中：
app.config.ts:
{
  "myPlugin": {
    "provider": "wxAPPID",
    "version": "1.0.0",
    "genericsImplementation": {
      "list": {
        "mp-view": "components/comp-from-miniprogram"
      }
    }
  }
}

页面调用插件页面：
<Navigator url="plugin://myPlugin/list">
  <Button>跳转到插件页面</Button>
</Navigator>
点击按钮跳转时小程序报错Component is not found in path "components/comp-from-miniprogram"，页面路径肯定没有问题，taro版本是3.5.5，如果我要在插件页渲染小程序组件还有其它方法吗？

**Comment by touristzn** [comment]
插件页面有调用

**Comment by winddies** [comment]
+1, 也遇到这个问题了
